### PR TITLE
Create Attendance card

### PR DESCRIPTION
**Pull request by annalbirena**
# Related Issues
- #14 

# Propsed Changes
- Add Attendance card

# Additional Info
- [Visx](https://airbnb.io/visx) library for Pie chart

### CODE REVIEW

File: src/components/pages/dashboard/Charts/attendancePie.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    console.log(item);
+    pieItems.push({
+      id: item.id,
+      label: item.course,
+      porcentage: 100 * (item.days_attended / totalDays),
+      color: item.color,
+      font_color: '#fff',
+    });
+    return <></>;
+  });
```
Comment (arinchaik) [inline src/components/pages/dashboard/Charts/attendancePie.tsx:47]: i have a suggestion you should not map this const in a render scope, maybe keep it on useStatte and use useEffect to set first state

**Review by arinchaik — COMMENTED** [review]
the suggestion about useState and useEffect

**Comment by Chandraprakash-Darji** [comment]
@arinchaik  is there anyway to check how the webpage look after this changes without pushing it

**Comment by arinchaik** [comment]
@Chandraprakash-Darji you mean to view this changes without merge?

**Comment by Chandraprakash-Darji** [comment]
@arinchaik  Means when we make an pull request vercel deploy to preview channel in presonal repo but how to make it here in organization repo. Main brach should be deployed on main domain and pull request code on preview channel

**Comment by arinchaik** [comment]
I'm not sure about the fork project but in my organization repo i have authorization in both of versel and github repo that make me have access to deploy preview on versel for any branch

**Comment by arinchaik** [comment]
@Chandraprakash-Darji  for now if you want to view this webpage changes you need to clone her fork project and view it on the current pushing branch

**Comment by arinchaik** [comment]
Screen Shot 2565-10-10 at 17 58 10

all about UI look good, I think you can merge this pr 
but if you concern about my comment i will improve for you soon

**Comment by Chandraprakash-Darji** [comment]
@arinchaik  Thanks for sharing this

**Comment by Chandraprakash-Darji** [comment]
@annalbirena  resolve the conflict and then i aprove it

**Comment by Chandraprakash-Darji** [comment]
@arinchaik  there is another pull request having changes in `yarn.lock` how can undo that.

**Comment by Chandraprakash-Darji** [comment]
@arinchaik I tried but it is not delteting the file

**Comment by annalbirena** [comment]
@Chandraprakash-Darji I fixed it

**Comment by Chandraprakash-Darji** [comment]
@annalbirena  Solve the yarn.lock conflict

**Comment by annalbirena** [comment]
Conflict resolved.

**Comment by Chandraprakash-Darji** [comment]
@annalbirena Add comments to the attendace pie component

**Comment by Chandraprakash-Darji** [comment]
I will finally merge it

**Comment by Chandraprakash-Darji** [comment]
@annalbirena  test are not passed solve the error

**Comment by annalbirena** [comment]
@Chandraprakash-Darji Hi! please review the updates

**Comment by Chandraprakash-Darji** [comment]
@arinchaik Can you help with this?

**Review by Chandraprakash-Darji — APPROVED** [review]

**Review by Chandraprakash-Darji — APPROVED** [review]

**Pull request merged**